src/main/java/teammates/storage/search/SearchManager.java

```diff
@@ -93,10 +94,11 @@ public static void putDocuments(String indexName, List<Document> documents) {
         Index index = getIndex(indexName);
 
         int delay = 2;
-        for (int attempts = 0; attempts < MAX_RETRIES; attempts++) {
+        boolean isSuccessful = false;
+        for (int attempts = 0; attempts < MAX_RETRIES && !isSuccessful; attempts++) {
             try {
                 List<Document> failedDocuments = putDocuments(index, documents);
```

Comment:
This is not as clear and would result in a checkstyle complaint (reassignment of parameter).